packages/security/src/scanning/ScanResultProcessor.ts

```
@@ -0,0 +1,480 @@
+  /**
+   * Validate format-specific data
+   */
+  private async validateFormatSpecificData(
+    result: ScanResult,
+  ): Promise<ValidationResult> {
+    switch (result.format) {
+      case 'encrypted_export':
+        const encryptedValidation = v.safeParse(
+          EncryptedExportDataSchema,
+          result.data,
+        );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)